### ISSUE TITLE
NTR: Adenovirus PCR in blood or sputum positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8091]
New term request Adenovirus PCR in blood or sputum positive (nadea3_190227154005): Adeno positive polymerase chain reaction; DNA  UMLS: 4763709

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000739
- label: Positive bloodstream adenovirus nucleic acid test
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of an adenovirus in the blood circulation by a methiod such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8091
- synonyms:
